be/src/common/config.h

```diff
@@ -179,6 +179,8 @@ CONF_mInt32(doris_scanner_queue_size, "1024");
 CONF_mInt32(doris_scanner_row_num, "16384");
 // single read execute fragment row bytes
 CONF_mInt32(doris_scanner_row_bytes, "10485760");
+// one scanner_thread with multiple rowsets use one rowblockv2
+CONF_mBool(enable_rowblockv2_reuse, "true");
```

Comment:
Why do we need this config iterm?

Reply:
if there are some bug，we can roll back by setting this config to false

Reply:
ok, I will remove it